### DISCUSSION TITLE
프로그램 업데이트 시 계정 정보를 불러오는 방법

### DISCUSSION THREAD

**Discussion by byungmeo** [byungmeo/GersangStation#27] [Announcements]
![image](https://user-images.githubusercontent.com/37506142/213318026-729f7732-d4c2-41b8-a6a3-633c10a9e163.png)
프로그램 업데이트 후 최초 실행 시 다음과 같은 창이 뜹니다.
"예" 버튼을 누르면 이전 설정 파일을 불러올 수 있습니다.

![image](https://user-images.githubusercontent.com/37506142/213317654-b0c48f02-1d8a-4ecd-b61b-a8b6638766b2.png)
![image](https://user-images.githubusercontent.com/37506142/213318261-790a3912-cdda-4b6c-89dc-d0a39c5702b6.png)
![image](https://user-images.githubusercontent.com/37506142/213318431-3c417e60-c182-4eb7-84d4-f5675d9a6c09.png)

**Comment by qogus00** [comment]
어떤 이유인지 모르겠으나 1.6.2 버전에서 기존 컨피그를 불러올 때 '오류 창이 팝업' 됩니다만,
닫고 재실행 하면 정상적으로 잘 불려와 집니다.

**Reply by byungmeo** [reply]
아마 새로운 설정값(심볼릭 방식 사용 여부) 추가하면서 발생한 문제같은데 정확히 어떤 오류인지 확인되면 고치도록 하겠습니다 감사합니다

**Comment by qogus00** [comment]
어머 친절도 하셔라 ^^ 즐거운 연말 보내셔요~
ᐧ

2025년 12월 3일 (수) PM 8:05, byungmeo ***@***.***>님이 작성:
